presto-cli/src/main/java/com/facebook/presto/cli/ErrorMessages.java

```diff
@@ -88,7 +90,12 @@ public Builder addTip(Tips tip, Object ...toFormat)
 
             public String build()
             {
-                return builder.toString();
+                if (!session.isQuiet()) {
```

Comment:
`--debug` + `--quiet` will enable technical details but disable tips. I didn't want to spend to much time thinking about other possible solutions for that as that seemed to be good enough.

:+1: TIP_INTRO
:+1: use `quiet` as `debug`